docs/rules/no-constant-condition.md

```diff
@@ -75,6 +75,10 @@ do {
 var result = x !== 0 ? a : b;
 ```
 
+See Also:
```

Comment:
Our convention is to have a related rules section at the bottom of documentation. See https://eslint.org/docs/rules/no-ternary#related-rules